features/unique_email.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Unique Emails
+In order to identify my users via e-mail addresses (for example during login processes)
```

Comment:
Order is wrong?

Reply:
No, this is the default generated by features, user stories have another order.

Reply:
please, don't start with the `In order to` - line. `As a *` should be the beginning of this description, for better readability